AIPscan/Reporter/report_format_versions_count.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from flask import render_template, request
+
+from AIPscan.helpers import parse_date
+from AIPscan.Data import data
+from AIPscan.Reporter import reporter, translate_headers, request_params
+
+
+@reporter.route("/report_format_versions_count/", methods=["GET"])
+def report_format_versions_count():
+    """Return overview of format versions in Storage Service."""
+    storage_service_id = request.args.get(request_params["storage_service_id"])
+    start_date = parse_date(request.args.get(request_params["start_date"]))
+    end_date = parse_date(request.args.get(request_params["end_date"]))
+
+    version_data = data.format_versions_count(
+        storage_service_id=storage_service_id, start_date=start_date, end_date=end_date
+    )
+    versions = version_data.get(data.FIELD_FORMAT_VERSIONS)
+
+    headers = [
+        data.FIELD_PUID,
+        data.FIELD_FORMAT,
+        data.FIELD_VERSION,
+        data.FIELD_COUNT,
+        data.FIELD_SIZE,
+    ]
+
+    return render_template(
+        "report_format_versions_count.html",
+        storage_service_id=storage_service_id,
+        storage_service_name=version_data.get(data.FIELD_STORAGE_NAME),
+        columns=translate_headers(headers),
+        versions=versions,
+        total_file_count=sum(version.get(data.FIELD_COUNT, 0) for version in versions),
+        total_size=sum(version.get(data.FIELD_SIZE, 0) for version in versions),
```

Comment:
I'm getting a stack trace error here:
```
Traceback (most recent call last):
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask_restx/api.py", line 639, in error_router
    return original_handler(e)
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ross-spencer/git/artefactual-labs/aipscanvenv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ross-spencer/git/artefactual-labs/AIPscan/AIPscan/Reporter/report_format_versions_count.py", line 37, in report_format_versions_count
    total_size=sum(version.get(data.FIELD_SIZE, 0) for version in versions),
```
It's not coming to me tonight why I might be getting `None` but I am wondering if you have an instinct? We might also need to capture that in tests?

Reply:
I will add the check for `None` in the Data endpoint here, but agree that it would be good to capture this in tests! I will work on including that in my revisions.